Modify Title bar / Header bar color with window focus. Nemo improvements. Typo Fix.

smurphos opened the pull request.

actionless commented on src/gtk-3.20/scss/apps/_nemo.scss:
```diff
@@ -18,7 +18,8 @@
     .nemo-window {
         .nemo-inactive-pane .view { 
-            background-color: mix($base_color, $text_color, .12);
+            background-color: darken($base_color, 5%); 
```
darken is not working fine with both light and dark colorscheme, use mix instead

actionless commented:
@jdamboeck could you try this on your setup?

actionless commented on src/gtk-3.20/scss/_global.scss:
```diff
@@ -81,6 +81,7 @@ $toolbar_bg_color: $bg_color;
 $toolbar_fg_color: $fg_color;
 
 $titlebar_bg_color: $dark_bg_color;
+$titlebar_bg_uf_color: mix($bg_color, $dark_bg_color, .6);
```
also i would like it to be named `titlebar_bg_unfocused` (as wm_bg, wm_border and wm_title above)

smurphos commented:
Hi,

Sorry for the delay in actioning the review request. So I've amended titlebar_bg_color to titlebar_bg_focused, and titlebar_bg_unfocused and checked and replaced for instances of titlebar_bg_color elsewhere.

Reverted to mix for the Nemo dual pane as requested.

actionless commented on src/gtk-3.20/scss/_global.scss:
```diff
@@ -105,23 +105,16 @@ $lightdm_fg_color: $dark_fg_color;
 
-$wm_bg: $titlebar_bg_color;
+$wm_bg_focused: $titlebar_bg_focused;
+$wm_bg_unfocused: mix($bg_color, $titlebar_bg_unfocused, .1);
```
i think the error is because of mix here, i'll try to debug it

actionless commented on src/metacity-1/metacity-theme-3.xml:
```diff
@@ -1,15 +1,16 @@
 <metacity_theme>
 <info>
-	<name>Numix</name>
-	<author>Satyajit Sahoo</author>
```
i think your changes are not big enough to discard Satya's authorship on the original theme

actionless commented on src/gtk-3.20/scss/_global.scss:
```diff
@@ -62,25 +62,25 @@ $switch_bg_color: mix($bg_color, $base_color, .3);
 $switch_fg_color: $text_color;
 $switch_slider_bg_color: mix($text_color, $bg_color, .5);
 
-//$link_color: mix($selected_bg_color, $fg_color, .3);
```
why you decided to randomly remove it?

actionless commented on src/gtk-3.20/scss/widgets/_button.scss:
```diff
@@ -346,7 +346,7 @@ $button_border_width: 1px;
         //color: mix($bg, $fg, .2);
         color: $fg;
-        border-color: alpha($button_bg, 0);
+        @include border(alpha($fg, .1 + $button_border_strength));
```
also some weird random change

actionless commented on src/gtk-3.20/scss/widgets/_base.scss:
```diff
@@ -35,7 +35,7 @@
 
     &:disabled,
-    &:disabled:disabled { color: mix($fg_color, $bg_color, .5); }
+    &:disabled:disabled { color: $link_color; }
```
how disabled style related to link color?

actionless commented on src/gtk-3.20/scss/_global.scss:
```diff
@@ -62,25 +62,25 @@ $switch_bg_color: mix($bg_color, $base_color, .3);
 
-$titlebar_bg_color: $dark_bg_color;
+$titlebar_bg_focused: $dark_bg_color;
+$titlebar_bg_unfocused: mix($bg_color, $dark_bg_color, .6);
```
i think the error is because of mix here, i'll try to debug it

actionless commented on src/gtk-3.20/scss/_global.scss:
```diff
@@ -62,25 +62,25 @@ $switch_bg_color: mix($bg_color, $base_color, .3);
 $switch_slider_bg_color: mix($text_color, $bg_color, .5);
 
-//$link_color: mix($selected_bg_color, $fg_color, .3);
-$link_color: #%TERMINAL_COLOR4%;
```
why you decided to randomly remove it?

smurphos commented:
Sorry, I'm going to start this PR afresh. Just made the mistake of using the same branch to merge into my own branch in my fork with other changes that I didn't intend to commit here.

I'll keep my Github practice to my own repos in future. Again sorry.

actionless commented:
no problem, wil be waiting for new PR!
